.travis.yml

```diff
@@ -22,7 +22,7 @@ addons:
 
 install:
   # Install ansible
-  - pip install ansible ansible-lint molecule[podman]
+  - pip install ansible ansible-lint molecule[docker] molecule-podman
```

Comment:
Side note: I can't believe that this travis reconfig worked the first time.

Reply:
The default scenario still uses molecule podman, and in molecule v3, this is a separate package to molecule itself.
Eventually, we'll need to update to using podman, and for things like healthcheck (although this would be in tripleo-collectd), there's a difference in the implementation IIUC between podman and docker.